src/Template/Bake/Template/index.ctp

```diff
@@ -16,7 +16,9 @@
 <?php
 /**
   * @var \<%= $namespace %>\View\AppView $this
+<% if ($entityClass): %>
```

Comment:
The entity interface makes total sense 👍 
Once the actual entity exists, the IdeHelper will replace the annotation then automatically anyway.